service/lib/agama/software/proposal.rb

```diff
@@ -153,12 +153,8 @@ def languages=(value)
 
       # Initializes the target, closing the previous one
       def initialize_target
-        Yast::Pkg.TargetFinish # ensure that previous target is closed
-        Yast::Pkg.TargetInitialize(Yast::Installation.destdir)
-        Yast::Pkg.TargetLoad
-
         preferred, *additional = languages
-        Yast::Pkg.SetPackageLocale(preferred) if preferred
+        Yast::Pkg.SetPackageLocale(preferred || "")
```

Comment:
Always set the locale, "" means no locale for libzypp. After removing the wrong target re-initialization above we need to reset the previously set locale if the current one is not defined.